packages/astro/src/core/endpoint/index.ts

```diff
@@ -85,6 +85,7 @@ export function createAPIContext({
 				return currentLocale;
 			}
 			if (locales) {
+				// @ts-ignore
```

Comment:
Use `ts-expect-error` instead. Also, since we own the code base, it's not good we need to suppress our errors. This is a  symptom that something isn't right

Reply:
This file is not referenced by anything.
- #10324